code/modules/client/preferences.dm

```diff
@@ -1735,6 +1738,19 @@ var/const/MAX_SAVE_SLOTS = 10
 					user?.client.tgui_say?.load()
 					save_preferences()
 
+				if("customcursors")
+					var/result = tgui_alert(user, "Do you want custom cursors enabled?", "Custom Cursors", list("Yes", "No"))
+					if(!result)
+						return
+					if(result == "Yes")
+						custom_cursors = TRUE
+						to_chat(src, SPAN_NOTICE("You're now using custom cursors."))
+					else
+						custom_cursors = FALSE
+						to_chat(src, SPAN_NOTICE("You're no longer using custom cursors."))
+					save_preferences()
+
```

Comment:
for maintainability reasons shouldn't this use a proc, since it's used in multiple places?